sdk/core/core-rest-pipeline/documentation/pipeline.md

```diff
@@ -0,0 +1,251 @@
+
+### Policies included in a default pipeline
+
+- proxyPolicy <img src="https://img.shields.io/badge/node.js-6DA55F?style=for-the-badge&logo=node.js&logoColor=white" alt="NodeJS Only" title="NodeJS Only" width="60"/>
+  - A policy that allows one to apply proxy settings to all requests. If not passed static settings, they will be retrieved from the HTTPS_PROXY or HTTP_PROXY environment variables.
+- decompressResponsePolicy <img src="https://img.shields.io/badge/node.js-6DA55F?style=for-the-badge&logo=node.js&logoColor=white" alt="NodeJS Only" title="NodeJS Only" width="60"/>
+  - A policy to enable response decompression according to Accept-Encoding header
+- formDataPolicy
+  - A policy that encodes FormData on the request into the body.
+- userAgentPolicy
+  - A policy that sets the User-Agent header (or equivalent) to reflect the library version
+- setClientRequestIdPolicy
+  - The programmatic identifier of the setClientRequestIdPolicy
+- defaultRetryPolicy
+  - A policy that retries according to three strategies:
+    - When the server sends a 429 response with a Retry-After header.
+    - When there are errors in the underlying transport layer (e.g. DNS lookup failures).
+    - Or otherwise if the outgoing request fails, it will retry with an exponentially increasing delay.
+- tracingPolicy
+  - A simple policy to create OpenTelemetry Spans for each request made by the pipeline that has SpanOptions with a parent. Requests made without a parent Span will not be recorded.
+- redirectPolicy
+  - A policy to follow Location headers from the server in order to support server-side redirection.
+- logPolicy
+  - A policy that logs all requests and responses.
+
+There are other policies commonly added to the default pipeline:
+
+- serializationPolicy
+  - This policy handles assembling the request body and headers using an OperationSpec and OperationArguments on the request.
+- deserializationPolicy
+  - This policy handles parsing out responses according to OperationSpecs on the request.
+
+# Authentication and Signing
+
+Pipeline policies are used to authenticate requests in Azure SDKs. An authentication policy would sign the request before sending it with the required authentication information, these policies run in the `Sign` phase.
+
+For example, the `bearerTokenAuthenticationPolicy` would sign the request by adding the bearer token in the request headers before the request is sent.
+
+# Configuring client pipeline
+
+When instantiating a new client a pipeline can be configured by adding additional policies to the default pipeline, passing a static array of policies.
+
+When configuring additional policies the following execution positions can be specified
+
+- `perCall` - When a policy is assigned a `perCall` position, it will be run once before sending the request. If the request needs to be retried the policy won't run again for each retry it will just be un once `perCall`.
```

Comment:
```suggestion
- `perCall` - When a policy is assigned a `perCall` position, it will be run once before sending the request. If the request needs to be retried, the policy will not be called again.
```